Chat-Ops-Collector/src/main/java/com/capitalone/dashboard/repository/GitHubRepoRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.capitalone.dashboard.repository;
+
+import com.capitalone.dashboard.model.GitHubRepo;
+import org.bson.types.ObjectId;
+import org.springframework.data.mongodb.repository.Query;
+
+import java.util.List;
+
+public interface GitHubRepoRepository extends BaseCollectorItemRepository<GitHubRepo> {
```

Comment:
I thought I removed it, not sure why my fork did not get updated with my commit, as I used the Github collector as a template